lib/router.js

```diff
@@ -42,7 +42,7 @@ router.applyMiddleware = function() {
     if (req.headers["content-type"]) {
       // 415 Unsupported Media Type
       if (req.headers["content-type"].match(/^application\/vnd\.api\+json;.+$/)) {
-        return res.status(415).end();
+        return res.status(415).end("HTTP 415 Unsupported Media Type - " + req.headers["content-type"]);
```

Comment:
Add some visible delimiter around `req.headers["content-type"]` above to spot whitespace easily if it exists at the end of the string?